packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -257,6 +267,15 @@ contract Migration is Script, UsingRegistry {
     );
   }
 
+  // TODO(Arthur): Add FeeCurrencyDirectory function
+  function migrateFeeCurrencyDirectory() public {
```

Comment:
As of https://github.com/celo-org/celo-monorepo/pull/10992/commits/99b59afbee3990557a54fcc1ed2539a70fd41678, the script (`run_integration_tests_in_anvil.sh`) fails with the following error:

```sh
$ forge --version && forge compile
$ ./migrations_sol/run_integration_tests_in_anvil.sh

# ...

Error:
Compiler run failed:
Error (9582): Member "initialize" not found or not visible after argument-dependent lookup in type(contract IFeeCurrencyDirectory).
   --> migrations_sol/Migration.s.sol:276:30:
    |
276 |       abi.encodeWithSelector(IFeeCurrencyDirectory.initialize.selector)
    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It originates in this function I wrote: 

https://github.com/celo-org/celo-monorepo/blob/35258dcd71a2f9036b5dde341dafae7407502f9e/packages/protocol/migrations_sol/Migration.s.sol#L271-L277

Reply:
The error makes sense, because there is no `initialize` function in `IFeeCurrencyDirectory.sol`, unlike `IFeeCurrencyWhitelist.sol`:

https://github.com/celo-org/celo-monorepo/blob/35258dcd71a2f9036b5dde341dafae7407502f9e/packages/protocol/contracts/common/interfaces/IFeeCurrencyWhitelist.sol#L5-L5

Instead, it's defined in `FeeCurrencyDirectory.sol`:

https://github.com/celo-org/celo-monorepo/blob/35258dcd71a2f9036b5dde341dafae7407502f9e/packages/protocol/contracts-0.8/common/FeeCurrencyDirectory.sol#L18-L20

Reply:
I was calling `initialize()` on  `IFeeCurrencyDirectory`, but that function is defined in `FeeCurrencyDirectory`.
This is fixed in https://github.com/celo-org/celo-monorepo/pull/10992/commits/a36eae5bc2c9c2071fb942199b1ba265f3937c17.